File: app/src/main/java/com/babestudios/companieshouse/data/model/CompanySearchResult.java
package com.babestudios.companieshouse.data.model;

import java.util.ArrayList;
import java.util.List;

import com.google.gson.annotations.SerializedName;

public class CompanySearchResult {

	@SerializedName("total_results")
	public Integer totalResults;

	@SerializedName("items")
	public List<CompanySearchResultItem> items = new ArrayList<CompanySearchResultItem>();

}

File: data/src/main/java/com/babestudios/companyinfo/data/model/search/CompanySearchResult.kt
package com.babestudios.companyinfo.data.model.search

import java.util.ArrayList

import com.google.gson.annotations.SerializedName

class CompanySearchResult {

	@SerializedName("total_results")
	var totalResults: Int? = null

	@SerializedName("items")
	var items: MutableList<CompanySearchResultItem> = ArrayList()

}
